### PR TITLE
Update Min Vid Experiment page for queue feature release

### PR DESCRIPTION
**Pull request by meandavejustice**
The image referenced in the new details section has not been uploaded yet.

It needs to be modified to have a gradient border/background like the other screenshots.

Here is a copy of the image:
min-vid-queue-view

cc/ @johngruen

### CODE REVIEW

**Comment by johngruen** [comment]
detail4

**Comment by johngruen** [comment]
detail4

File: content-src/experiments/min-vid.yaml [hunk 1]
```diff
@@ -6,16 +6,23 @@ description: >
...
+  is a media player and the other is the entire Web.</p>
   <ul>
     <li><strong>Click and go:</strong> Pop up the Min Vid frame to keep videos
       playing in the foreground while you browse.</li>
     <li><strong>Put it where you want it:</strong> Min Vid lets you drag the
       video frame anywhere on your desktop.</li>
     <li><strong>Controls where you need them:</strong> Min Vid includes controls
       to let you pause/play, scrub, drag, minimize, and adjust volume right in the frame.</li>
+    <li><strong>Now supporting audio playback:</strong> Min Vid now supports playback from
+      Soundcloud and direct mp3 links.</li>
```
Comment (ghost) [inline content-src/experiments/min-vid.yaml]: Soundcloud should be capitalized:   SoundCloud

File: content-src/experiments/min-vid.yaml [hunk 2]
```diff
@@ -6,16 +6,23 @@ description: >
...
+    <li><strong>Create a queue:</strong> Min Vid now supports queues, just start up
+      a video and add tracks from any supported domains (YouTube, Vimeo, Soundcloud, and
+      direct links)</li>
```
Comment (ghost) [inline content-src/experiments/min-vid.yaml]: Should have a period

File: content-src/experiments/min-vid.yaml [hunk 3]
```diff
@@ -6,16 +6,23 @@ description: >
...
+    <li><strong>View your history:</strong> Min Vid keeps track of your history, just pop
+    open the queue panel from the top right of the player and you can access all the media
+    you've been playing.'</li>
```
Comment (ghost) [inline content-src/experiments/min-vid.yaml]: stray apostrophe

File: content-src/experiments/min-vid.yaml [hunk 4]
```diff
@@ -42,15 +49,20 @@ details:
   -
     image: /static/images/experiments/min-vid/experiments_experimentdetail/detail1.jpg
     copy: >
-      Access Min Vid from YouTube and Vimeo video players.
+      Access Min Vid from YouTube, Vimeo, and Soundcloud players.
```
Comment (ghost) [inline content-src/experiments/min-vid.yaml]: capitalize again

**Comment by SoftVision-CiprianMuresan** [comment]
@meandavejustice,
With this PR closed closed, I assumed that the Test Pilot website would be updated with the latest changes. But it seems to me like some things are still missing:
- 3 out of 4 pictures are out of date and have no mention of Queue (one even has the drag handle and another has the outdated "Send to mini player" context menu option)
- 2 out of 4 pictures from the onboarding tour are out of date 

Will the images and onboarding tour be updated?

**Pull request merged**